src/Type/Schema.php

```diff
@@ -373,6 +373,10 @@ private function defaultTypeLoader(string $typeName) : ?Type
      */
     public static function resolveType($type) : Type
     {
+        if ($type instanceof Type) {
+            return $type;
+        }
+
         if (is_callable($type)) {
             return $type();
         }
```

Comment:
The last return in this method will never happen (dead code)

_github wont allow me to comment L384_

Reply:
I guess the check for `is_callable()` can be removed. I believe it is pretty expensive, since many things in PHP qualify as callable, removing it should minimize the amount of work and speed up the benchmarks.

Reply:
SGTM 4666f7f

```
\GraphQL\Benchmarks\HugeSchemaBench

    benchSchema.............................I0 [μ Mo]/r: 161.827 161.827 (ms) [μSD μRSD]/r: 0.000ms 0.00%
    benchSchemaLazy.........................I0 [μ Mo]/r: 0.027 0.027 (ms) [μSD μRSD]/r: 0.000ms 0.00%
    benchSmallQuery.........................I0 [μ Mo]/r: 191.393 191.393 (ms) [μSD μRSD]/r: 0.000ms 0.00%
    benchSmallQueryLazy.....................I0 [μ Mo]/r: 182.581 182.581 (ms) [μSD μRSD]/r: 0.000ms 0.00%

\GraphQL\Benchmarks\LexerBench

    benchIntrospectionQuery.................I4 [μ Mo]/r: 7.081 7.103 (ms) [μSD μRSD]/r: 0.088ms 1.24%

\GraphQL\Benchmarks\StarWarsBench

    benchSchema.............................I1 [μ Mo]/r: 0.623 0.623 (ms) [μSD μRSD]/r: 0.011ms 1.72%
    benchHeroQuery..........................I1 [μ Mo]/r: 3.119 3.119 (ms) [μSD μRSD]/r: 0.012ms 0.39%
    benchNestedQuery........................I1 [μ Mo]/r: 7.002 7.002 (ms) [μSD μRSD]/r: 0.003ms 0.04%
    benchQueryWithFragment..................I1 [μ Mo]/r: 7.885 7.885 (ms) [μSD μRSD]/r: 0.014ms 0.17%
    benchStarWarsIntrospectionQuery.........I1 [μ Mo]/r: 64.344 64.345 (ms) [μSD μRSD]/r: 0.278ms 0.43%
```

Reply:
I don't know why phpbench still sometimes takes 161ms for that HugeSchemaBench run; if you look at the blackfire profile, you can see that even with 26570 calls, Schema::resolveType only takes up 12.9 ms.